contracts/test/MockRocketPool.sol

```diff
@@ -115,13 +116,13 @@ contract MockRocketPool is MultiRolesAuthority, ERC20Mintable {
 
     function getSharesByPooledEth(
```

Comment:
Hmm, do we even need these functions? Looks like this was copied from `MockLido`.

Reply:
probably not, definitely copied from there